### PR TITLE
update logs cpu/memory

### PR DESCRIPTION
**Pull request by SherinV**
Signed-off-by: Sherin Varughese <shvarugh@redhat.com>

**Related Issue:**   https://issues.redhat.com/browse/ACM-3041

### CODE REVIEW

**Review by jlpadilla — APPROVED** [review]
/lgtm
/hold

This PR will merge to main and fast-forward to release-2.8.  Good to merge without additional approvals.

**Comment by jlpadilla** [comment]
/cherry-pick release-2.7

This will create the PR once this is merged. We have to decide if we merge for 2.7.0 or 2.7.1

File: controllers/create_rolesbindings.go
Comment (SherinV) [inline controllers/create_rolesbindings.go:53]: Should we remove %s here? @xrajesh

File: controllers/create_collectordeployment.go
Comment (SherinV) [inline controllers/create_collectordeployment.go:83]: Should this be logged as error so that we can see the error?
log.Error(err, "Could not set control for search-collector deployment")

Comment (xrajesh) [reply controllers/create_collectordeployment.go:83]: We can skip logging the error .

File: controllers/create_rolesbindings.go
Comment (xrajesh) [reply controllers/create_rolesbindings.go:53]: yes thats a good idea , please push the change @SherinV

**Review by xrajesh — APPROVED** [review]

**Pull request merged**
